packages/@react-native-windows/cli/src/runWindows/runWindows.ts

```diff
@@ -54,16 +54,17 @@ function getPkgVersion(pkgName: string): string {
   return '';
 }
 
-enum RunWindowsPhase {
-  None,
-  AutoLink,
-  FindBuildTools,
-  NuGetRestore,
-  FindSolution,
-  Deploy,
-}
+/**
+ * Labels used by telemtry to represent current operation
+ */
+type RunWindowsPhase =
```

Comment:
Since we are changing the telemetry here, should this part be backproted to 0.63?

Reply:
btw: I prefer this style of enum as it is more js-style

Reply:
The strings should be the same as before, just missing a phase. (I also missed that we were translating enum to enum name before).